tests/components/openuv/test_sensors.py

```diff
@@ -0,0 +1,54 @@
+"""Test OpenUV sensors."""
+
+from operator import attrgetter
+
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.components.openuv.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr, entity_registry as er
+from homeassistant.helpers.entity_component import async_update_entity
+
+from .conftest import TEST_LATITUDE, TEST_LONGITUDE
+
+
+async def test_sensors(
+    hass: HomeAssistant,
+    set_time_zone,
+    config,
+    config_entry,
+    setup_config_entry,
+    snapshot: SnapshotAssertion,
+    device_registry: dr.DeviceRegistry,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test all sensors created by the integration."""
+    device_id = f"{TEST_LATITUDE}_{TEST_LONGITUDE}"
+    device = device_registry.async_get_device({(DOMAIN, device_id)})
+
+    assert device is not None
+    assert device == snapshot(name="device")
```

Comment:
device snapshot is something for in `test_init.py` and not for this PR preferably. I think we can pick this PR for the beta, so ideally we keep it small